doc/unison-manual.tex

```diff
@@ -183,69 +189,16 @@
-steps should get you a fully working installation in a few minutes.  If
-you run into trouble, you may find the suggestions on the
-\SHOWURL{http://www.cis.upenn.edu/\home{bcpierce}/unison/faq.html}{Frequently Asked
-Questions page} helpful.  Pre-built binaries are available for a
-variety of platforms.
-
-Unison can be used with either of two user interfaces:
-\begin{enumerate}
-\item a simple textual interface, suitable for dumb terminals (and
-running from scripts), and
-\item a more sophisticated graphical interface, based on Gtk2 (on
-       Linux/Windows) or the native UI framework (on OSX).
-\end{enumerate}
-
-You will need to install a copy of Unison on every machine that you
-want to synchronize.  However, you only need the version with a
-graphical user interface (if you want a GUI at all) on the machine
-where you're actually going to display the interface (the \CLIENT{}
-machine).  Other machines that you synchronize with can get along just
-fine with the textual version.
-
-
-\SUBSECTION{Downloading Unison}{download}
-
-See \ONEURL{https://github.com/bcpierce00/unison/wiki/Downloading-Unison}.
-
-If a pre-built binary of Unison is available for the client machine's
-architecture, just download it and put it somewhere in your search
-path (if you're going to invoke it from the command line) or on your
-desktop (if you'll be click-starting it).
-
-The executable file for the graphical version (with a name including
-\verb|gtkui|) actually provides {\em both} interfaces: the graphical one
-appears by default, while the textual interface can be selected by including
-\verb|-ui text| on the command line.  The \verb|textui| executable
-provides just the textual interface.
-
-If you don't see a pre-built executable for your architecture, you'll
-need to build it yourself.  See \sectionref{building}{Building Unison from Scratch}.
-
-Check to make sure that what you have downloaded is really executable.
-Either click-start it, or type \showtt{unison -version} at the command
-line.
-
-Unison can be used in three different modes: with different directories on a
-single machine, with a local or a remote machine over a direct socket connection, or
-with a remote machine using {\tt ssh} for authentication and secure
-transfer.
-
-\SUBSECTION{Running Unison}{afterinstall}
 
-Once you've got Unison installed on at least one system, read
-\sectionref{tutorial}{Tutorial} of the user manual (or type \showtt{unison -doc
-  tutorial}) for instructions on how to get started.
+\SECTION{Tutorial}{tutorial}{tutorial}
 
+%\finish{Put a pointer somewhere in here to the typical profile in the
+%  reference section.}
 
 \SUBSECTION{Upgrading}{upgrading}
 
-Upgrading to a new version of Unison is as simple as throwing away the old
-binary and installing the new one.
+(This section is perhaps misplaced, but is early because it is far
+better to have at least skimmed it than to not know it exists.)
```

Comment:
Agree, it feels misplaced. Perhaps make it a top-level section?

At some point (but not yet) we can remove all the text about compatibility and interoperability. Then the entire section becomes very short, or can be removed entirely.

Reply:
top it is